starlingcaptureapi/claim.py

```diff
@@ -100,7 +100,7 @@ def generate_create(self, jwt_payload, data):
             signature["data"] = signature_data
             assertions.append(signature)
 
-        timestamp = self._timestamp_from_meta(meta)
+        timestamp = self._get_value_from_meta(meta, "Timestamp")
```

Comment:
I guess in theory there could be a data bundle that has a timestamp but not lat/lon, but let's assume they always come in a bundle!